### PR TITLE
git-mailinfo: add page

### PR DESCRIPTION
**Pull request by Waples**
- [x] The page (if new), does not already exist in the repo.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

Ref: #3953

### CODE REVIEW

**Review by bl-ue — COMMENTED** [review]
Darn, I keep hitting Cmd+Enter which comments instead of approves 🙄

File: pages/common/git-mailinfo.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
+# git mailinfo
+
+> Extract patch and authorship information from a single email message.
+> More information: <https://git-scm.com/docs/git-mailinfo>.
+
+- Extract the patch and author data from an email message:
+
+`git mailinfo {{message|patch}}`
+
+- Extract but remove leading and trailing whitespace:
```
Comment (marchersimon) [inline pages/common/git-mailinfo.md:10]:
```suggestion
- Extract but remove leading and trailing whitespaces:
```

Comment (bl-ue) [reply pages/common/git-mailinfo.md:10]: This isn't really correct — whitespace is plural and singular.

Comment (marchersimon) [reply pages/common/git-mailinfo.md:10]: Oh, good to know.

**Review by marchersimon — APPROVED** [review]
Thanks @Waples

File: pages/common/git-mailinfo.md [hunk 2]
```diff
@@ -0,0 +1,16 @@
+# git mailinfo
+
+> Extract patch and authorship information from a single email message.
+> More information: <https://git-scm.com/docs/git-mailinfo>.
+
+- Extract the patch and author data from an email message:
+
+`git mailinfo {{message|patch}}`
```
Comment (sbrl) [inline pages/common/git-mailinfo.md:8]: Is this a path to a file, or something else? Same for the other examples.

Comment (Waples) [reply pages/common/git-mailinfo.md:8]: `mailinfo` is very obscure. It's one of those `git` functions that's not very well documented and I haven't been able to test it out properly yet.

**Review by CleanMachine1 — APPROVED** [review]
:+1:

**Pull request merged**